app/javascript/modCenter/moderationArticles.jsx

```diff
@@ -7,35 +7,57 @@ export class ModerationArticles extends Component {
       document.getElementById('mod-index-list').dataset.articles,
     ),
     prevSelectedArticleId: undefined,
-    selectedArticleId: undefined,
   };
 
-  toggleArticle = (id, path) => {
+  toggleArticle = (id, title, path) => {
     const { prevSelectedArticleId } = this.state;
     const selectedArticle = document.getElementById(`article-iframe-${id}`);
+    const selectedDetailsPanel = document.querySelector(
+      `details[data-testid='mod-article-${id}']`
+    );
 
     if (prevSelectedArticleId > 0) {
-      document.getElementById(
-        `article-iframe-${prevSelectedArticleId}`,
-      ).innerHTML = '';
+      if (
+        selectedDetailsPanel.getAttribute("open") !== null
+      ) {
+        if (prevSelectedArticleId !== id) {
+          const previousOpenDetailsPanel = document.querySelector(
+            `details[data-testid='mod-article-${prevSelectedArticleId}']`
+          );
+        
+          if (previousOpenDetailsPanel.getAttribute("open") !== null) {
+            previousOpenDetailsPanel.removeAttribute("open")
+          }
+        }
```

Comment:
```suggestion
        if (prevSelectedArticleId !== id) {
          document.querySelector(
            `details[data-testid='mod-article-${prevSelectedArticleId}']`
          )?.removeAttribute("open");
        }
```

refactor suggestion - we could make use of optional chaining to avoid having so many if statements in this block. The `removeAttribute` function will only get called if the previousOpenDetailsPanel exists, and if the "open" attribute isn't present, `removeAttribute` will just return without error 🙂